ansible_collections/arista/avd/roles/eos_designs/docs/input-variables.md

```diff
@@ -1140,6 +1068,14 @@ Note: structured configuration is not merged recursively and will be taken direc
 roles/eos_designs/docs/tables/svi-profiles.md
 --8<--
 
+### EVPN VLAN aware bundles settings
+
+Optional VLAN aware bundles to share common settings for l2vlans which are supposed to use the same vlan-aware-bundle.
+
+--8<--
+roles/eos_designs/docs/tables/evpn_vlan_bundles.md
```

Comment:
Please use hyphens in the table names instead of underscore.

Reply:
changed